.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+## Description
+
+Please include a summary of the change and which issue is fixed (if any) and any relevant motivation / context. List any dependencies that are required for this change. 
+Screenshots and / or videos can also be helpful if appropriate.
+
+Fixes # (issue)
+
+Please also tag the relevant team to help with review. You can tag any of the following:
+@paperless-ngx/backend (Python / django, database, etc.)
+@paperless-ngx/frontend (JavaScript/Typescript, HTML, CSS, etc.)
+@paperless-ngx/ci-cd (GitHub Actions, deployment)
+@paperless-ngx/test (General testing for larger PRs)
+
+## Type of change
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+
```

Comment:
What about a `Conducted tests` section. It allows

* ... to verify that the developer tested his feature with manual interactions
* ... helps the reviewer to understand the feature and how he can test it

Example: 
* Started paperless-ngx
* Added a document to the consume folder
* Checked that the imported document has XZY present

Reply:
Thanks for all the suggestions, I added some with slight adjustments here and there. The conducted tests one is only one Im not sure about. my worry is we dont want to make PRs overly-complicated such that it will put people off. Not sure what others think

Reply:
No problem. I have no strong opinion on that one. But from my code review experience so far, I can tell that it helped me a lot to know how the feature can be tested.

If you like we can omit the `Conducted tests` section for now and merge it as it is. When we feel like we need explanations on how those features can be tested or want to verify developers tested their features, we can still bring this back on the table. What do you think?

Reply:
Sounds good. Actually I will also add a note to the "description" section, youre right it would be important to be able to test something so hopefully a reminder is enough.